ingeniamotion/fsoe_master/safety_functions.py

```diff
@@ -102,6 +105,20 @@ class SafetyFunction:
     ios: dict[SafetyFieldMetadata, FSoEDictionaryItem]
     parameters: dict[SafetyFieldMetadata, SafetyParameter]
 
+    def activated_by(
+        self, _handler: "FSoEMasterHandler", _pdu_maps: "PDUMaps"
+    ) -> Optional["SafetyFunction"]:
+        """Get the safety function that activates this function, if any.
+
+        Args:
+            _handler: The FSoE master handler to use.
```

Comment:
marking _unused makes no sense to me. I can imagine the static analyzer complaining on the abstractmethod though.

Having a reference of the handle on each safety function would be okay for me, as the instances cant be reused across diferent handlers.

Passing the pdu maps i assume is because the wizard proposal has not been applied to the handler yet?